AI content updates (#17994)
ethereum/ethereum-org-website: public/content/ai-agents/payments/index.md

---
title: Machine payments with x402
description: How AI agents pay for compute, data, and API calls using the x402 protocol on Ethereum and layer 2 networks
lang: en
template: ai-agents
emoji: ":money_with_wings:"
sidebarDepth: 2
image: /images/ai-agents/hero-image.png
alt: Machine payments with x402 on Ethereum

> summaryPoint1: x402 enables per-request machine-to-machine payments using the HTTP 402 status code with stablecoin settlement

Reviewer: Trim to ≤8 words; keep keywords; parallel structure.

```suggestion
summaryPoint1: Per-request machine-to-machine payments using HTTP 402 and stablecoins
```

> summaryPoint1: x402 enables per-request machine-to-machine payments using the HTTP 402 status code with stablecoin settlement

Reviewer: Trim to ≤8 words; keep keywords; parallel structure.

```suggestion
summaryPoint1: Per-request machine-to-machine payments using HTTP 402 and stablecoins
```

> summaryPoint2: Agents pay for exactly what they use with no subscription, no account, and no human involvement after initial funding

Reviewer: Dropped "after initial funding" — true but a footnote-level nuance.

```suggestion
summaryPoint2: Pay-per-use, with no accounts, subscriptions, or human approval
```

> summaryPoint2: Agents pay for exactly what they use with no subscription, no account, and no human involvement after initial funding

Reviewer: Dropped "after initial funding" — true but a footnote-level nuance.

```suggestion
summaryPoint2: Pay-per-use, with no accounts, subscriptions, or human approval
```

> summaryPoint3: Production-ready with growing enterprise adoption across cloud, payments, and infrastructure providers

Reviewer: Dropped "infrastructure" since "cloud" already covers that category.

```suggestion
summaryPoint3: Production-ready, with adoption across cloud and payments providers
```

> summaryPoint3: Production-ready with growing enterprise adoption across cloud, payments, and infrastructure providers

Reviewer: Dropped "infrastructure" since "cloud" already covers that category.

```suggestion
summaryPoint3: Production-ready, with adoption across cloud and payments providers
```
